manoseimas/settings/base.py

```diff
@@ -157,6 +158,7 @@
     'manoseimas.compat',            # depends on: solutions
     'manoseimas.widget',
     'manoseimas.lobbyists',
+    'manoseimas.compatibility_test',
```

Comment:
Can you add a simple test that loads your test page, like:
```python
class WebpackIntegrationTestCase(TestCase):
    def test_webpack_loading(self):
        self.client.get('/test')
```
Just to check that webpack loader works with tests? You will probably need to add webpack building step to the test setup (something like `webpack -d -c path_to_config.js` before the tests are run should work).